sdks/python/apache_beam/transforms/core.py

```diff
@@ -304,7 +304,7 @@ def _fn_takes_side_inputs(fn):
   return len(argspec.args) > 1 + is_bound or argspec.varargs or argspec.keywords
 
 
-class CallableWrapperDoFn(DoFn):
+class CallableWrapperDoFn(NewDoFn):
```

Comment:
Yeah there will be two more PRs:
- Rename DoFn to OldDoFn
- Rename NewDoFn to DoFn